infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/storedprocedures/InfinispanRemoteNamedParametersStoredProcedureScriptTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.infinispanremote.test.storedprocedures;
+
+import java.util.List;
+import javax.persistence.EntityManager;
+import javax.persistence.ParameterMode;
+import javax.persistence.PersistenceException;
+import javax.persistence.StoredProcedureQuery;
+
+import org.hibernate.ogm.backendtck.storedprocedures.Car;
+import org.hibernate.ogm.utils.TestForIssue;
+import org.hibernate.ogm.utils.jpa.OgmJpaTestCase;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.fest.assertions.Assertions.assertThat;
+import static org.hibernate.ogm.backendtck.storedprocedures.Car.RESULT_SET_PROC_ID_PARAM;
+import static org.hibernate.ogm.backendtck.storedprocedures.Car.RESULT_SET_PROC_TITLE_PARAM;
+import static org.hibernate.ogm.backendtck.storedprocedures.Car.UNIQUE_VALUE_PROC_PARAM;
+
+/**
+ * Testing call of stored procedures
+ *
+ * @author The Viet Nguyen &amp;ntviet18@gmail.com&amp;
+ */
+@TestForIssue(jiraKey = { "OGM-1430" })
+public class InfinispanRemoteNamedParametersStoredProcedureScriptTest extends OgmJpaTestCase {
```

Comment:
Please, use test from 'core' module. All initializations must me in 'prepareDatabase' method

Reply:
Thank you, I forgot to removed skipped tests. They are enabled and passing now.